style.css

```diff
@@ -44,6 +72,7 @@ header {
 main {
     margin: 0 2rem;
     padding-bottom: 8rem;
+    /* flex: 1 0 */
 }
 
```

Comment:
Please can you remove `padding: 1rem;` from the `#spotify` selector? Because it's causing the spotify section to have more padding than the YouTube and Apple sections.

```
#spotify {
    padding: 1rem;
    display: flex;
    flex-direction: column;
}
```